WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderViewPagerTransformer.java

```diff
@@ -11,16 +11,16 @@
  * zoom & depth are based on examples here, with many fixes and simplifications:
  * http://developer.android.com/training/animation/screen-slide.html#pagetransformer
  */
-class ReaderViewPagerTransformer implements ViewPager.PageTransformer {
-    enum TransformType {
+public class ReaderViewPagerTransformer implements ViewPager.PageTransformer {
```

Comment:
We should move this class to  `Widgets` (or `Utils`) and call it `WPViewPagerTransformer`.